doc/architecture/system-and-security-architecture.rst

```diff
@@ -435,7 +435,8 @@ use them only in secure environments.
 
 The Ostro Project provides different pre-compiled images. All of them
 are compiled as development images. Currently there are no pre-compiled
-production images.
+production images. Those pre-compiled image get security updates
```

Comment:
Let's merge these short sentences to: 
The Ostro Project provides pre-compiled development images that can receive security updates using the software update mechanism.  No pre-compiled production images are provided.